Update franchise.yml

kevbentz opened the pull request:
Removed:

Law & Order: Hate Crimes

Star Wars: Rangers of the New Republic
Star Wars: Lando

Series have been cancelled/shelved and have not/will not be airing.  Listings have been removed from TMDB.

## Description

Please include a summary of the changes.

### Issues Fixed or Closed

- Fixes #(issue)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change (non-code changes affecting only the wiki)
- [ ] Infrastructure change (changes related to the github repo, build process, or the like)

## Checklist

- [x] My code was submitted to the nightly branch of the repository.

kevbentz committed:
Update franchise.yml

Removed:

Law & Order: Hate Crimes

Star Wars: Rangers of the New Republic
Star Wars: Lando

Series have been cancelled/shelved and have not/will not be airing.  Listings have been removed from TMDB.

YozoraXCII commented:
The worded lists after the IDs still mention the items you have removed - can you amend?

kevbentz committed:
Update franchise.yml

Removed:

Law & Order: Hate Crimes

Star Wars: Rangers of the New Republic
Star Wars: Lando

Series have been cancelled/shelved and have not/will not be airing. Listings have been removed from TMDB.

kevbentz commented:
Updated, thank you!
